diff --git a/common.gypi b/common.gypi
index ea08e803..fa94c9f4 100644
--- a/common.gypi
+++ b/common.gypi
@@ -189,7 +189,7 @@
     'msvs_settings': {
       'VCCLCompilerTool': {
         'StringPooling': 'true', # pool string literals
-        'DebugInformationFormat': 3, # Generate a PDB
+        'DebugInformationFormat': 0, # Generate a PDB
         'WarningLevel': 3,
         'BufferSecurityCheck': 'true',
         'ExceptionHandling': 0, # /EHsc
diff --git a/deps/cares/common.gypi b/deps/cares/common.gypi
index 609ad62a..d714cdd7 100644
--- a/deps/cares/common.gypi
+++ b/deps/cares/common.gypi
@@ -80,7 +80,7 @@
     'msvs_settings': {
       'VCCLCompilerTool': {
         'StringPooling': 'true', # pool string literals
-        'DebugInformationFormat': 3, # Generate a PDB
+        'DebugInformationFormat': 0, # Generate a PDB
         'WarningLevel': 3,
         'BufferSecurityCheck': 'true',
         'ExceptionHandling': 1, # /EHsc
diff --git a/deps/uv/common.gypi b/deps/uv/common.gypi
index ec482340..807de0aa 100644
--- a/deps/uv/common.gypi
+++ b/deps/uv/common.gypi
@@ -93,7 +93,7 @@
     'msvs_settings': {
       'VCCLCompilerTool': {
         'StringPooling': 'true', # pool string literals
-        'DebugInformationFormat': 3, # Generate a PDB
+        'DebugInformationFormat': 0, # Generate a PDB
         'WarningLevel': 3,
         'BufferSecurityCheck': 'true',
         'ExceptionHandling': 1, # /EHsc
diff --git a/src/node.h b/src/node.h
index 596769a6..21dbc38d 100644
--- a/src/node.h
+++ b/src/node.h
@@ -433,7 +433,7 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
 #ifdef NODE_SHARED_MODE
 # define NODE_CTOR_PREFIX
 #else
-# define NODE_CTOR_PREFIX static
+# define NODE_CTOR_PREFIX
 #endif
 
 #if defined(_MSC_VER)
